docs/examples/driver_examples/Qcodes example with keithley 2450.ipynb

```diff
@@ -0,0 +1,457 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "keithley.sweep.setup(0, 10E-6, 10)\n",
+    "keithley.output(True)\n",
+    "data = Measure(keithley.sweep).run()\n",
+    "keithley.output(False)\n",
+    "MatPlot(data.arrays['keithley_sweep'])"
```

Comment:
Matplot is never imported anywhere?

Reply:
I was still in the process of cleaning up this example file. I will use the data set everywhere in the next iteration